x-pack/plugins/fleet/common/services/agentless_policy_helper.ts

```diff
@@ -5,6 +5,47 @@
  * 2.0.
  */
 
+import type { PackageInfo, RegistryPolicyTemplate } from '../types';
+
+export const isAgentlessIntegration = (
```

Comment:
It recently came up that we don't have a way to specify which policy template to check.  For example, the [CSPM integration](https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/manifest.yml#L89) has multiple policy templates (KSPM, CSPM, CNVM) however only CSPM supports agentless.

Would it be possible to have an optional parameter that we can pass `cspm` and check individual policy templates?